src/main/java/com/exasol/dbbuilder/dialects/AbstractSchemaChild.java

```diff
@@ -11,16 +11,13 @@
     /**
      * Create a new child object of a database schema.
      *
-     * @param writer       database object writer to use
      * @param parentSchema parent schema
      * @param name         name of the database object
      * @param owned        {@code true} if the object is owned by the TDDB, {@code false} if the TDDB attached to a
      *                     database object that already existed
-     *
      */
-    public AbstractSchemaChild(final DatabaseObjectWriter writer, final Schema parentSchema, final String name,
-            final boolean owned) {
-        super(writer, name, owned);
+    public AbstractSchemaChild(final Schema parentSchema, final Identifier name, final boolean owned) {
```

Comment:
```suggestion
    public abstract AbstractSchemaChild(final Schema parentSchema, final Identifier name, final boolean owned) {
```

:-)

Reply:
Wait, but it's an implemented constructor. Why should it be abstract? How do I save fields?

Reply:
Sorry, I read over it to fast and the class definition was collapsed. I thought `abstract` was missing in the class definition. Forget that comment.